howtos/how-to-shard-a-conversation/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>TalkJS - Shard a Conversation</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet">
+</head>
+
+<body>
+    <script>
+        (function (t, a, l, k, j, s) {
+            s = a.createElement("script");
+            s.async = 1;
+            s.src = "https://cdn.talkjs.com/talk.js";
+            a.head.appendChild(s);
+            k = t.Promise;
+            t.Talk = {
+                v: 3,
+                ready: {
+                    then: function (f) {
+                        if (k)
+                            return new k(function (r, e) {
+                                l.push([f, r, e]);
+                            });
+                        l.push([f]);
+                    },
+                    catch: function () {
+                        return k && new k();
+                    },
+                    c: l,
+                },
+            };
+        })(window, document, []);
+    </script>
+    <script src="index.js" async defer></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js"
```

Comment:
Is Popper something you're using with Bootstrap? Would be good to mention it in the tutorial if it's required.